### ISSUE TITLE
Missing dependencies: "filter"

### ISSUE THREAD

**Issue by pomkos** [causefx/Organizr#1192]
###### Operating System: FreeBSD
<hr>

##### Problem Description:
Vague "dependency missing" message. All dependencies are checked off, except for "filter". I'm not sure what that dependency is exactly?

<hr>

##### Reproduction Steps:
Taken from the Organizr part of this link:

https://www.ixsystems.com/community/resources/fn11-2-iocage-jails-plex-tautulli-sonarr-radarr-lidarr-jackett-transmission-organizr.58/

Basically installed Organizr, nginx, php, etc in a FreeBSD virtual machine.

<hr>

#### Errors on screen?  If so paste here:

```
Dependencies Missing

* filter
```

**Comment by Roxedus** [comment]
Its php-filter, or php7-filter depending on your package repo

**Comment by tronyx** [comment]
Yep, what @si0972  said. All you have to do is install it within the Jail and then restart the Jail.

**Comment by pomkos** [comment]
Oh dang, that was easy enough. Thanks a lot to both of you!

For future users:

I just had to execute ` iocage exec organizr pkg install php72-filter` and then restart the jail from the FreeNAS GUI

**Comment by tronyx** [comment]
Thanks for the instructions. Please close at your convenience as the issue has been resolved.

**Comment by causefx** [comment]
Eventually i need to write some notes on this stuff....  At least the error caught it as this only affects FreeBSD
